src/main/java/co/fairtide/prometheus_aeron_stat/PrometheusAeronStat.java

```diff
@@ -0,0 +1,263 @@
+//                        return;
+                }
+            }
+        }
+        if (portNumber == -1) {
+            System.out.println("Port to expose prometheues statistics is a compulsory argument");
+            System.out.println(
+                    "Usage: [port=<port for prometheus server COMPULSORY>]" +
+                            "\t[delay=<seconds between updates>]%n" +
+                            "\t[-Daeron.dir=<directory containing CnC file>] ");
+            System.exit(0);
+            return;
+        }
+
+        HTTPServer prometheusServer = new HTTPServer(portNumber);
+
+        final PrometheusAeronStat prometheusAeronStat = new PrometheusAeronStat(mapCounters());
+        final AtomicBoolean running = new AtomicBoolean(true);
+        SigInt.register(() -> running.set(false));
+
+        while (running.get()) {
+            prometheusAeronStat.updatePrometheus();
+            Thread.sleep(delayMs);
+        }
+// TODO Add archive logs when martin thompson fixes the bug https://github.com/real-logic/aeron/issues/565
+//        try (AeronArchive archive = AeronArchive.connect()) {
+//            while (running.get()) {
+//                prometheusAeronStat.updatePrometheus();
+//                prometheusAeronStat.printArchiveLogs(archive);
+//                Thread.sleep(delayMs);
+//            }
+//
+//        } catch (Exception e) {
+//            System.out.println(e);
+//        }
+        prometheusServer.stop();
+
+    }
+
+
+    public void updatePrometheus() {
+        this.clearOldCounter();
+        counters.forEach(
+                (counterId, typeId, keyBuffer, label) ->
+                {
+                    switch (typeId) {
+                        case 0:
+                            this.updateSystemCounter(counters, counterId, typeId, label);
+                            break;
+                        case 1:
+                            this.numChannelPublishingCounter++;
+                            break;
+                        case 2:
+                            break;
+                        case 3:
+                            break;
+                        case 4:
+                            this.numSubscriberCounter++;
+                            break;
+                        case 5:
+                            break;
+                        case 6:
+                            break;
+                        case 7:
+                            break;
+                        case 9:
+                            break;
+                        case 10:
+                            break;
+                        case 11:
+                            System.out.println(label);
+                            this.numClientCounter++;
+                            break;
+                        default:
+                            break;
+                    }
+                });
+        System.out.println("Lol: " + this.numChannelPublishingCounter);
```

Comment:
What is **Lol** here? Is it some sort of debug logging? Can you either remove it or change the message?

Reply:
Btw, consider using a logging framework instead of printing messages directly to STDIN

Reply:
That was a accident that i push that up. I dont think i will use a logging frame work here.